packages/compass-e2e-tests/tests/oidc.test.ts

```diff
@@ -318,21 +358,36 @@ describe('OIDC integration', function () {
       };
     };
 
+    if (TEST_MULTIPLE_CONNECTIONS) {
+      await browser.removeConnection(connectionName);
+      await browser.clickVisible(Selectors.Multiple.SidebarNewConnectionButton);
+      await browser.$(Selectors.ConnectionModal).waitForDisplayed();
+    }
     await browser.setValueVisible(
       Selectors.ConnectionFormStringInput,
       connectionString
     );
     await browser.clickVisible(Selectors.ConnectButton);
 
-    const modal = '[role=dialog]';
-    const cancelButton = 'button:nth-of-type(2)';
-    await browser.waitUntil(async () => {
-      const modalHeader = await browser.$(`${modal} h1`);
-      return (await modalHeader.getText()).includes('Authentication expired');
-    });
+    await browser.pause(10_000);
```

Comment:
Maybe a comment here that this should aligned with the token expiration or something